datadog_sync/model/logs_pipelines.py

```diff
@@ -48,18 +50,42 @@ async def pre_apply_hook(self) -> None:
         self.destination_integration_pipelines = await self.get_destination_integration_pipelines()
 
     async def create_resource(self, _id: str, resource: Dict) -> Tuple[str, Dict]:
+        destination_client = self.config.destination_client
+
         if resource["is_read_only"]:
             if resource["name"] not in self.destination_integration_pipelines:
-                raise SkipResource(
-                    _id,
-                    self.resource_type,
-                    "integration pipelines cannot be created only updated. "
-                    + f"Skipping sync. Enable integration pipeline {resource['name']}",
-                )
+                # Extract the source from the query
+                source = resource.get("filter", {}).get("query", "").split("source:")[-1]
+                if not source:
+                    raise Exception(f"Source not found in the query for integration pipeline '{resource['name']}'")
+                payload = {
+                    "ddsource": source,
+                    "ddtags": ",".join(DEFAULT_TAGS),
+                    "message": f"[datadog-sync-cli] Triggering creation of '{resource['name']}' integration pipeline",
+                }
+
+                # Submit a log to the logs intake API to trigger the creation of the integration pipeline
+                await destination_client.post(self.logs_intake_path, payload, subdomain=self.logs_intake_subdomain)
+                created = False
+                for _ in range(3):
```

Comment:
NIT: I think it's probably safer to wait longer on this, it can take time to ingest this log, I'm thinking that we can make it 1 minutes (12 * 5 seconds)